### PR TITLE
Add scitran.py targets, and adapt to changes in API

### PR DESCRIPTION
**Pull request by kevlarkevin**
The `uwsgi` command is now given a bunch of args, instead of a configuration file.  The fig.yml template was updated accordingly, and the production.ini has been completely removed. 

The fig and nginx configuration has been updated to use two ports (80, 443), instead of three.

I also changed `scitran.py` script to make it easier (IMO) to add new targets.

### CODE REVIEW

**Comment by kevlarkevin** [comment]
Just added some changes to default configurations for nginx and uwsgi.  Important one is uwsgi arg (--socket_timeout) to make sure uwsgi doesn't close sockets during long-running batch downloads.

**Comment by kevlarkevin** [comment]
And... the downloadable release.tar has been updated.

File: scitran.py [hunk 1]
```diff
@@ -163,136 +251,133 @@ def getTarball(name):
...
+
     # Detect if cluster is new (has never been started before)
-    newCluster = not os.path.isfile(os.path.join('data', 'mongo', 'mongod.lock'))
+    newCluster = not os.path.isfile(os.path.join('persistent', 'mongo', 'mongod.lock'))
     email = ""
-
-    # If cluster is brand new, run bootstrap instance
     if newCluster:
         print "\nIt looks like this is a new scitran instance. Would you like to add some data? "
         email = raw_input("If so, enter a valid " + config['auth']['provider'] + " email for your first user: ").strip()
```
Comment (kofalt) [inline scitran.py]: When trying out this branch, I think there's a regression with detecting new cluster state. After starting a new cluster (with all new state, due to renaming `data` -> `persistent`), and running the start target a second time, it asks me for bootstrap info again:

```
./scitran.py start

(re)starting the instance
Copying key+cert.pem into api and nginx bind mount locations

It looks like this is a new scitran instance. Would you like to add some data? 
If so, enter a valid Google email for your first user:
```

Comment (kevlarkevin) [inline scitran.py]: I have not been able to reproduce.  I start a fresh installation, and get the guided prompts and then bootstrapping occurs.  I rerun start, and the cluster goes up without prompts. I run stop, then start, and the cluster goes up without prompts.

Can you check two things:  Did all three containers actually started? Does `/persistent/mongo/mongod.lock` actually exist?

Comment (kofalt) [inline scitran.py]: After internal discussion, this was an out-of-date code issue on our vendored folders :/

Works fine now.

**Comment by kofalt** [comment]
While running the script:

> Starting a service container...

Is this accurate? I don't see one generated, I thought we dropped that entirely.

**Comment by kofalt** [comment]
As per internal discussion, we don't want to store the CWD in the config file for two reasons:

1. By definition, if you can find the config file you know the folder ;)
1. A scitran install is fundamentally portable - stop the cluster, tar it up, move to another machine.

I added a commit that uses the CWD implicitly and does not save it to the config file.

**Pull request merged**